gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
```

Comment:
Can you do a local `./gradlew clean` or something to get the wrapper to generate the gradlew scripts. Typically they also update.

Reply:
Nothing changes apart from that file.